sashimi-webapp/src/database/create/entitiesCreator.js

```diff
@@ -1,22 +1,11 @@
-import constants from '../constants';
-import tableCreator from './tableCreator';
-import exceptions from '../exceptions';
-import initDataGenerator from './initDataGenerator';
-import SqlCommands from '../sql-related/sqlCommands';
+import constants from 'src/database/constants';
+import tableCreator from 'src/database/create/tableCreator';
+import exceptions from 'src/database/exceptions';
+import initDataGenerator from 'src/database/create/initDataGenerator';
+import SqlCommands from 'src/database/sql-related/sqlCommands';
 
 const sqlCommands = new SqlCommands();
 
-// a promise to allow sequential running of keying in data
-function initFillUpDefaultDatas() {
-  if (typeof Promise === 'function') {
-    return new Promise((resolve, reject) => {
-      resolve(true);
-    });
-  } else {
-    throw new exceptions.PromiseFunctionNotDefined();
-  }
-}
-
 function isTableExists(tableName) {
   if (typeof Promise === 'function') {
```

Comment:
oh ok. yeah, most of the functions can be called without calling `const something = new ClassFunction()`. and noted thanks